test/autests/gold_tests/body_verification/http.yaml

```diff
@@ -0,0 +1,140 @@
+meta:
+  version: '1.0'
+sessions:
+- protocol:
+  - name: http
+    version: 1.1
+  - name: tls
+    sni: test_sni
+  - name: tcp
+  - name: ip
+    version: 4
+  transactions:
+  - all:
+      headers:
+        fields:
+        - [uuid, 1]
+    client-request:
+      method: POST
+      scheme: https
+      url: /a/path
+      version: '1.1'
+      headers:
+        fields:
+        - [Host, example.data.com]
+        - [Content-Type, text/html]
+        - [Content-Length, '11']
+      content:
+        encoding: plain
+        data: client_test
+        size: 11
+
+    proxy-request:
+      content:
+        encoding: plain
```

Comment:
Are these `encoding` nodes used in `proxy-request:content`? I suspect they can just be removed. The `proxy-*` nodes should just be used in tests like this for verification. (In Traffic Dump, they are useful to record what the proxy saw, but that's a different use case.)

Reply:
I have these here for the new format I implemented for the body verification, since the `data` node can also be used to  specify the value, the `encoding` node is also kept for proper value encoding.

Reply:
Makes sense. 👍